src/components/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="app">
+    <template v-if="$store.state.user">
+      <h2>You are logged in!</h2>
+      <div class="logout">
+        <button @click="logout">Logout</button>
+      </div>
+      <router-view></router-view>
+    </template>
+    <template v-else>
+      <div class="login">
+        <Auth />
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import Auth from './Auth';
```

Comment:
Ok this is where we call in the whole authentication suite from the Auth component, which gives us the ability to use <Auth /> in the template to show aspects of the UI (for example)